lib/unionlabs/Cargo.toml

```diff
@@ -22,7 +22,7 @@ frame-support-procedural = { workspace = true }
 generic-array            = { workspace = true }
 hex                      = { workspace = true }
 hex-literal              = { workspace = true }
-k256                     = { version = "0.13.1", default-features = false, features = ["schnorr"] } # needs to be brought in explicitly since bip32 doesn't expose k256/schnorr
+k256                     = { version = "0.13.1", default-features = false, features = ["schnorr", "alloc"] } # needs to be brought in explicitly since bip32 doesn't expose k256/schnorr
```

Comment:
why was this required?

Reply:
The `to_vec` function of signature seemed to be locked behind it.